ci-operator/config/medik8s/machine-deletion-remediation/medik8s-machine-deletion-remediation-main.yaml

```diff
@@ -20,9 +24,10 @@ operator:
 releases:
   latest:
     candidate:
+      architecture: amd64
       product: ocp
-      stream: ci
-      version: "4.10"
+      stream: nightly
+      version: "4.11"
```

Comment:
Why only OCP v4.11?

Reply:
That was a mistake, I'll revert back to 4.10